build.gradle.kts

```diff
@@ -17,12 +26,20 @@ buildscript {
     gradlePluginPortal()
   }
   dependencies {
+<<<<<<< Updated upstream
+=======
+    classpath(libs.pluginPackages.grgit)
+    classpath(libs.pluginPackages.jetbrains.changelog)
+    classpath(libs.pluginPackages.jetbrains.grammarkit)
+    classpath(libs.pluginPackages.jetbrains.kotlin)
+>>>>>>> Stashed changes
     classpath(libs.pluginPackages.taskTree)
     classpath(libs.pluginPackages.versionCatalogUpdate)
     classpath(libs.pluginPackages.versionsFilter)
   }
 }
 
+apply<ChangelogPlugin>()
```

Comment:
Actually... now having a second thought that for the plugins that we don't need on buildSrc compile classpath i.e. `changelog`, `taskTree`, `versionCatalogUpdate` and `versionsFilter`, it makes more sense to use the classic plugin application after all. No need to deal with `apply<...Plugin>()` etc.